platform/ui/src/components/StudyListPagination/StudyListPagination.tsx

```diff
@@ -33,7 +35,7 @@ const StudyListPagination = ({
         <div className="flex justify-between">
           <div className="flex items-center">
             <Select
-              id={"rows-per-page"}
+              id={'rows-per-page'}
```

Comment:
maybe instead we should follow REact/JSX good practices and do not use curly braces for static content such strings
I.e
id="rows-per-page"